src/components/TestimonialsSection.jsx

```diff
@@ -34,6 +34,9 @@ const TestimonialsSection = () => {
 
   useEffect(() => {
     timeoutActive && startTimer()
+    return () => {
+      stopTimer()
+    }
```

Comment:
This fucks up the testimonial loading loop.
If 37-39 is removed the userCanSelectLanguage.feature.js pass 👌
Would you like to inform me why it was put here in the first place?

Reply:
it's a clean up function